web/src/components/tasks/LicenseTask.tsx

```diff
@@ -184,6 +184,7 @@ export const LicenseTask = (props: Props): ReactElement => {
                   status={licenseStatusResult.status}
                   items={licenseStatusResult.checklistItems}
                   onEdit={onEdit}
+                  agency={props.task.agencyId ?? ""}
```

Comment:
Curious why agency is required and potentially passed in with an `"""` but context is optional. I think it should be one or the other not both